pycobertura/cobertura.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, xml_path, base_path=None):
 
     @memoize
     def _get_element_by_class_name(self, class_name):
-        syntax = "./packages/package/classes/class[@name='%s'][1]" % class_name
+        syntax = "./packages/package/classes/class[@filename='%s'][1]" % class_name
```

Comment:
The tests fail because this is not PEP-8 compliant. Can you please wrap around the line like this?

```python
    @memoize
    def _get_element_by_class_name(self, class_name):
        syntax = "./packages/package/classes/class[@filename='%s'][1]" % (
            class_name
        )
        return self.xml.xpath(syntax)[0]
```